Fix build errors seen with glibc 2.20

/home/ubuntu/work/upstream/openembedded-core/build/tmp-glibc/sysroots/imx53qsb/usr/include/xorg/os.h:579:1: error: expected identifier or '(' before '__extension__'
 strndup(const char *str, size_t n);
 ^

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending
Index: xserver-xorg-video-imx-11.09.01/src/imx_accel.c
===================================================================
--- xserver-xorg-video-imx-11.09.01.orig/src/imx_accel.c	2011-07-26 14:22:47.000000000 -0700
+++ xserver-xorg-video-imx-11.09.01/src/imx_accel.c	2014-08-29 15:54:43.648070587 -0700
@@ -21,6 +21,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
  * SOFTWARE.
  */
+#include <xorg-server.h>
 
 #include <stdint.h>
 #include "xf86.h"
Index: xserver-xorg-video-imx-11.09.01/src/imx_exa_offscreen.c
===================================================================
--- xserver-xorg-video-imx-11.09.01.orig/src/imx_exa_offscreen.c	2014-08-29 15:26:31.992070587 -0700
+++ xserver-xorg-video-imx-11.09.01/src/imx_exa_offscreen.c	2014-08-29 15:54:54.632070587 -0700
@@ -53,7 +53,7 @@
  * When allocating, the contiguous block of areas with the minimum eviction
  * cost is found and evicted in order to make room for the new allocation.
  */
-
+#include <xorg-server.h>
 
 #include "xf86.h"
 #include "exa.h"
Index: xserver-xorg-video-imx-11.09.01/src/imx_exa_z160.c
===================================================================
--- xserver-xorg-video-imx-11.09.01.orig/src/imx_exa_z160.c	2014-08-29 15:26:31.992070587 -0700
+++ xserver-xorg-video-imx-11.09.01/src/imx_exa_z160.c	2014-08-29 15:54:36.056070587 -0700
@@ -21,6 +21,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
  * SOFTWARE.
  */
+#include <xorg-server.h>
 
 #include "xf86.h"
 #include "xf86_OSproc.h"
Index: xserver-xorg-video-imx-11.09.01/src/imx_ext.c
===================================================================
--- xserver-xorg-video-imx-11.09.01.orig/src/imx_ext.c	2014-08-29 15:26:32.064070587 -0700
+++ xserver-xorg-video-imx-11.09.01/src/imx_ext.c	2014-08-29 15:55:18.048070587 -0700
@@ -21,6 +21,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
  * SOFTWARE.
  */
+#include <xorg-server.h>
 
 #include <X11/X.h>
 #include <X11/Xproto.h>
